.rubocop_todo.yml

```diff
@@ -1,19 +1,20 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config --auto-gen-only-exclude`
-# on 2020-11-03 23:10:27 UTC using RuboCop version 0.93.1.
+# on 2020-11-04 21:37:55 UTC using RuboCop version 0.93.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 2
+# Offense count: 3
 # Configuration parameters: IgnoredMethods, Max.
 Metrics/AbcSize:
   Exclude:
+    - 'app/controllers/works_controller.rb'
```

Comment:
I really wish we knew which method had this problem. Putting inline would help.

Reply:
I reckon that if we are shooting to have classes that are relatively small and single-purpose, it will be relatively easy to tell what the offending method is. I'm happy to try a refactoring to remove this exclusion altogether. I'd rather do that than do an inline exclusion.